gorm-tools/src/main/groovy/gorm/tools/repository/model/AbstractCrossRefRepo.groovy

```diff
@@ -240,11 +242,8 @@ abstract class AbstractCrossRefRepo<X, P extends Persistable, R extends Persista
      * This should NOT normally be called directly, use addOrRemove
      */
     List<X> addOrRemoveList(P main, List<Map> dataList){
-        //if its passing in an empty list on update then clear it out
```

Comment:
Lets talk about this. There was a good reason for doing this so that the rest api can clear lists and this may have a larger impact. i think this also has an impact on the rest docs. Where does this get used?

Reply:
see https://go.9ci.io/rndc/api/#examples
The default is documented as replace. and I went around on this and its the most common and logical default.